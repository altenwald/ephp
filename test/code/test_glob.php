<?php

var_dump(glob("test/*", GLOB_NOCHECK | GLOB_MARK | GLOB_ONLYDIR));
