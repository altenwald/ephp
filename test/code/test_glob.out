array(3) {
  [0]=>
  string(13) "test/classes/"
  [1]=>
  string(10) "test/code/"
  [2]=>
  string(14) "test/requests/"
}
